test/test_tokenizer.py

```diff
@@ -1714,6 +1864,245 @@ def test_abbrev() -> None:
         Tok(kind=TOK.S_END, txt=None, val=None),
     ]
 
+    tokens = list(t.tokenize("Sjö millj. krónur fóru í lagfæringarnar."))
+    tokens = strip_originals(tokens)
+    assert tokens == [
+        Tok(kind=TOK.S_BEGIN, txt=None, val=(0, None)),
+        Tok(kind=TOK.WORD, txt="Sjö", val=None),
+        Tok(
+            kind=TOK.WORD,
+            txt="millj.",
+            val=[
+                BIN_Tuple(
+                    stofn="milljónir",
+                    utg=0,
+                    ordfl="kvk",
+                    fl="skst",
+                    ordmynd="millj.",
+                    beyging="-",
+                ),
+                BIN_Tuple(
+                    stofn="milljarðar",
+                    utg=0,
+                    ordfl="kk",
+                    fl="skst",
+                    ordmynd="millj.",
+                    beyging="-",
+                ),
+            ],
+        ),
+        Tok(kind=TOK.WORD, txt="krónur", val=None),  # Should this be amount?
+        Tok(kind=TOK.WORD, txt="fóru", val=None),
+        Tok(kind=TOK.WORD, txt="í", val=None),
+        Tok(kind=TOK.WORD, txt="lagfæringarnar", val=None),
+        Tok(kind=TOK.PUNCTUATION, txt=".", val=(3, ".")),
+        Tok(kind=TOK.S_END, txt=None, val=None),
+    ]
+
+    tokens = list(t.tokenize("Aðgerðin kostaði níutíu þús.kr."))
+    tokens = strip_originals(tokens)
+    assert tokens == [
+        Tok(kind=TOK.S_BEGIN, txt=None, val=(0, None)),
+        Tok(kind=TOK.WORD, txt="Aðgerðin", val=None),
+        Tok(kind=TOK.WORD, txt="kostaði", val=None),
+        Tok(kind=TOK.WORD, txt="níutíu", val=None),
+        Tok(
+            kind=TOK.WORD,
+            txt="þús.kr.",
+            val=[BIN_Tuple("þúsundir króna", 0, "kvk", "skst", "þús.kr.", "-")],
+        ),
+        Tok(kind=TOK.S_END, txt=None, val=None),
+    ]
+
+    tokens = list(t.tokenize("Aðgerðin kostaði 14 þús.kr."))
```

Comment:
@sultur Can you please add a test that puts several spaces between 14 and þús.kr. and checks that the original text of the amount token correctly includes these spaces? Also, as another case, try "14þús.kr." without spaces, and that the original text is correct there also?

Reply:
Good idea! Added two tests for that.